quantum-viz/quantum_viz/utils.py

```diff
@@ -0,0 +1,71 @@
+import tempfile
+import warnings
+import webbrowser
+from pathlib import Path
+from typing import Optional
+from typing import Union
+
+from qiskit import QuantumCircuit
+from quantum_viz.qiskit_parser import qiskit2json
+
+HTML_TEMPLATE = """<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>quantum-viz.js visualization</title>
+</head>
+<body>
+  <div id="msg" style="font-size: large; font-weight: bolder;">
+    There was some error with the script
+  </div>
+  <div id="circuit_div"></div>
+  <script src="https://unpkg.com/@microsoft/quantum-viz.js{0}"></script>
+  <script type="text/javascript">
+  var circuitString = `{1}
+  `;
+  </script>
+  <script type="text/javascript">
+    var circuit = JSON.parse(circuitString);
+    if (typeof qviz != 'undefined' && typeof circuit != undefined) {{
+      document.getElementById('msg').style.display = 'none';
+      var displayDiv = document.getElementById('circuit_div');
+      if (displayDiv != null) {{
+        qviz.draw(circuit, displayDiv, qviz.STYLES['{2}']);
+      }}
+    }}
+  </script>
+
+</body>
+</html>
+"""
+SUFFIX = "_qviz.html"
+STYLES = ("Default", "BlackAndWhite", "Inverted")  # Use Literal type in python 3.8
+
+
+def display(
+    circuit: QuantumCircuit,
```

Comment:
I suggest to also support qviz JSON here, such that qiskit can be an optional dependency.